README.md

```diff
@@ -90,457 +90,467 @@ type: tutorial
-    - On successful request you can see a status: `201 Created` in your postman
-    6. Click on the `Headers` to view the response
-    7. You can see a `Location` header with a URL something like `eddi://ai.labs.regulardictionary/regulardictionarystore/regulardictionaries/<UNIQUE_DICTIONARY_ID>?version=<UNIQUE_DICTIONARY_VERSION>`. Make a note of the `<UNIQUE_DICTIONARY_ID>` as shown. 
+    
+        ```json
+        {
+            "words": [
+                {
+                    "word": "hello",
+                    "expressions": "greeting(hello)",
+                    "frequency": 0
+                },
+                {
+                    "word": "hi",
+                    "expressions": "greeting(hi)",
+                    "frequency": 0
+                },
+                {
+                    "word": "bye",
+                    "expressions": "goodbye(bye)",
+                    "frequency": 0
+                },
+                {
+                    "word": "take test",
+                    "expressions": "taketest(yes)",
+                    "frequency": 0
+                },
+                {
+                    "word": "thanks",
+                    "expressions": "thanks(thanks)",
+                    "frequency": 0
+                }
+            ],
+            "regExs": [],
+            "phrases": [
+                {
+                    "phrase": "good afternoon",
+                    "expressions": "greeting(good_afternoon)"
+                },
+                {
+                    "phrase": "how are you",
+                    "expressions": "how_are_you"
+                }
+            ]
+        }
+        ```
+    
+    6. Click the **Send** button to make the request. If your request is sucessful, you will see a status: `201 Created` in your postman.
+    7. Click on the `Headers` to view the response
+    8. You can see a `Location` header with a URL something like `eddi://ai.labs.regulardictionary/regulardictionarystore/regulardictionaries/<UNIQUE_DICTIONARY_ID>?version=<UNIQUE_DICTIONARY_VERSION>`. Make a note of the `<UNIQUE_DICTIONARY_ID>` as shown. 
 
-![](doc/source/images/regulardictionary.png)
+    ![](doc/source/images/regulardictionary.png)
 
->Note: The `<UNIQUE_DICTIONARY_ID>` will be used in Packaging, please make a note of it.
+>Note: The `<UNIQUE_DICTIONARY_ID>` will be used to package the elements of the chatbot, so please make a note of it.
 
-#### Step 4.2: Create Behavior Rules
+#### Step 4.2: Create behavior rules
 
-- Behavior Rules are Actions based on decision making with predefined as well as custom conditions.
+*Behavior rules* are actions that are based on decision-making with predefined as well as custom conditions. <!--EM: So, basically, behavior rules are rules that how the chatbot will respond to answers? or where they will look for answers or all of the above? When I read the current definition, I think: Whose decision-making?? the machines?-->
```

Comment:
**So, basically, behavior rules are rules that how the chatbot will respond to answers? or where they will look for answers or all of the above? When I read the current definition, I think: Whose decision-making?? the machines?**

Behavior rules are a series of defined rules by the developer of the bot. These rules are used to answer to the queries asked by the user.